src/css/components/_captions-settings.scss

```diff
@@ -21,6 +21,48 @@
   vertical-align: bottom;
 }
 
+// code that will only run if CSS Grid is supported by the browser
+@supports (display: grid) {
+  .vjs-text-track-settings .vjs-modal-dialog-content {
+    display: grid;
+    grid-template-columns: 1fr auto;
+    grid-template-rows: 1fr auto;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-colors {
+    display: block;
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-font {
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 2;
+    grid-row: 2;
+    text-align: unset;
```

Comment:
why are these two unset?

Reply:
not unsetting them keeps the buttons visually at the bottom right where they should be.

Reply:
I thought it gave it a weird 3 column effect. But i can revert this if you'd like.

Reply:
yeah, I think I'd prefer to keep it as is, we can change it up a bit later on if that was wanted.